### DISCUSSION TITLE
Error: package or namespace load failed for ‘Seurat’ in dyn.load

### DISCUSSION THREAD

**Discussion by lavalle17** [satijalab/seurat#4779] [Q&A]
Hi, I would really appreciate your help, im have problems when i  install seurat, i got the follow warnings and errors, i wonder if you can help me with this:

> library(Seurat)

Error: package or namespace load failed for ‘Seurat’ in dyn.load(file, DLLpath = DLLpath, ...):
 unable to load shared object '/usr/local/lib/R/site-library/igraph/libs/igraph.so':
  libxml2.so.2: cannot open shared object file: No such file or directory
In addition: Warning message:
package ‘Seurat’ was built under R version 4.0.5

**Comment by timoast** [comment]
I'd suggest reinstalling igraph and Seurat, and try again. If you still have problems please open a new issue.

**Comment by lillian863** [comment]
> Hi, I would really appreciate your help, im have problems when i install seurat, i got the follow warnings and errors, i wonder if you can help me with this:
> 
> > library(Seurat)
> 
> Error: package or namespace load failed for ‘Seurat’ in dyn.load(file, DLLpath = DLLpath, ...): unable to load shared object '/usr/local/lib/R/site-library/igraph/libs/igraph.so': libxml2.so.2: cannot open shared object file: No such file or directory In addition: Warning message: package ‘Seurat’ was built under R version 4.0.5

hello, Could I ask that you had resolved your question?

**Comment by lillian863** [comment]
> Hi, I would really appreciate your help, im have problems when i install seurat, i got the follow warnings and errors, i wonder if you can help me with this:
> 
> > library(Seurat)
> 
> Error: package or namespace load failed for ‘Seurat’ in dyn.load(file, DLLpath = DLLpath, ...): unable to load shared object '/usr/local/lib/R/site-library/igraph/libs/igraph.so': libxml2.so.2: cannot open shared object file: No such file or directory In addition: Warning message: package ‘Seurat’ was built under R version 4.0.5

hello, Could I ask that you had resolved your question?
